### PR TITLE
Chore/#162370317/create readme file

### PR DESCRIPTION
**Pull request by kleva-j**
#### What does this PR do?
- Creates a README.md file that show information about the app.

#### Description of Task to be completed?
- Create a readme file.
- Write a descriptive overview of the app

#### How should this be manually tested?
- Clone the repository
- Checkout into the branch Chore/#162370317/create-readme-file
- Install all dependencies
- run `npm start` to play the game successfully.

#### What are the relevent pivotal tracker stories?
- [162370317](https://www.pivotaltracker.com/story/show/162370317)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,5 +20,5 @@ before_script:
 script:
   - npm test
 after_success: 
-  - npm run coveralls
+  - npm run coverage
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```
Comment (fuzzybrows) [inline .travis.yml]: EOF missing. There should be an `End Of File` line added to every file

Comment (fuzzybrows) [reply .travis.yml]: This is a rule defined in the Airbnb rules

**Comment by fuzzybrows** [comment]
Generally looks okay, except for its missing a heroku link to test the endpoints

File: .travis.yml (hunk 1 above)
Comment (kleva-j) [reply .travis.yml]: I have added the `End of File` line to my travis.yml file.

Comment (kleva-j) [reply .travis.yml]: Please, can I go ahead and merge this branch?

**Pull request merged**